### PR TITLE
Add NERC-NEUTRONRCH7BT

### PR DESCRIPTION
**Pull request by DusKing1**

### CODE REVIEW

File: configs/default/NERC-NEUTRONRCH7BT.config [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+set baro_i2c_device = 2
+set blackbox_device = SDCARD
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 100
+set beeper_inversion = ON
+set beeper_od = OFF
+set sdio_use_4bit_width = ON
+set sdio_device = 1
+set debug_mode = DUAL_GYRO_SCALED
```
Comment (Asizon) [inline configs/default/NERC-NEUTRONRCH7BT.config:184]: Seems that you forgot remove it after your testtings :)

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/NERC-NEUTRONRCH7BT.config (hunk 1 above)
Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH7BT.config]: removed

**Review by Asizon — APPROVED** [review]

File: configs/default/NERC-NEUTRONRCH7BT.config [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+set sdio_device = 1
+set max7456_spi_bus = 2
+set pinio_config = 129,1,1,1
+set pinio_box = 0,40,41,42
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set gyro_1_align_yaw = 2700
+set gyro_2_bustype = SPI
+set gyro_2_spibus = 4
```
Comment (haslinghuis) [inline configs/default/NERC-NEUTRONRCH7BT.config:192]: Missing LF

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH7BT.config:192]: What is LF

Comment (Asizon) [reply configs/default/NERC-NEUTRONRCH7BT.config:192]: White line at the end of file

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH7BT.config]: fixed

**Review by haslinghuis — DISMISSED** [review]

File: configs/default/NERC-NEUTRONRCH7BT.config [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+# master
+set gyro_to_use = BOTH
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 2
+set blackbox_device = SDCARD
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 100
```
Comment (SteveCEvans) [inline configs/default/NERC-NEUTRONRCH7BT.config:179]: Why is this target specific? This value is dependent on the ESC current sensor, not the FC.

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH7BT.config]: fixed

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**